xarray/tests/test_coding_times.py

```diff
@@ -16,10 +16,12 @@
                requires_cftime_or_netCDF4, has_cftime, has_dask)
 
 
-_NON_STANDARD_CALENDARS = {'noleap', '365_day', '360_day',
-                           'julian', 'all_leap', '366_day'}
-_ALL_CALENDARS = _NON_STANDARD_CALENDARS.union(
-    coding.times._STANDARD_CALENDARS)
+_NON_STANDARD_CALENDARS_SET = {'noleap', '365_day', '360_day',
+                               'julian', 'all_leap', '366_day'}
+_ALL_CALENDARS = sorted(_NON_STANDARD_CALENDARS_SET.union(
+    coding.times._STANDARD_CALENDARS))
```

Comment:
These could be lists throughout (a single cast for `_STANDARD_CALENDARS`) with the union as a simple `+`, but this works fine